ds_judgements_public_ui/templates/includes/results_filters_inputs.html

```diff
@@ -1,93 +1,61 @@
-{% load court_utils errors %}
+{% load court_utils errors search_results_filters crispy_forms_tags %}
 <div class="structured-search__multi-fields-panel">
   <div class="structured-search__limit-to-container">
-    <fieldset class="{% if context.errors|has_errors_for_field:"from_date" %}with-errors{% endif %}">
+    <fieldset class="{% if form.non_field_errors %}with-errors{% endif %}">
```

Comment:
The `if` being inside the quotes irks me, but it looks like it's valid to have class=""